tests/revwalk/hidecb.c

```diff
@@ -123,7 +123,24 @@ void test_revwalk_hidecb__add_hide_cb_multiple_times(void)
 
 	cl_git_pass(git_revwalk_new(&walk, _repo));
 	cl_git_pass(git_revwalk_add_hide_cb(walk, hide_every_commit_cb, NULL));
-	cl_git_fail(git_revwalk_add_hide_cb(walk, hide_every_commit_cb, NULL));
+	cl_git_pass(git_revwalk_add_hide_cb(walk, hide_every_commit_cb, NULL));
+	cl_git_pass(git_revwalk_add_hide_cb(walk, NULL, NULL));
```

Comment:
Good. But you don't verify that the revwalk actually works (e.g. by calling `git_revwalk_next()`).

Reply:
Ah, gotcha. Updated again.